docs/02.API-REFERENCE.md

```diff
@@ -5838,6 +5838,60 @@ jerry_get_proxy_target (jerry_value_t proxy_value)
 
 - [jerry_create_proxy](#jerry_create_proxy)
 - [jerry_create_special_proxy](#jerry_create_special_proxy)
+- [jerry_get_proxy_handler](#jerry_get_proxy_handler)
+
+## jerry_get_proxy_handler
+
+**Summary**
+
+Gets the handler object of a Proxy object
+
+*Notes*:
+- This API depends on a build option (`JERRY_BUILTIN_PROXY`) and can be checked
+  in runtime with the `JERRY_FEATURE_PROXY` feature enum value,
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- The es.next profile enables this by default.
+
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_get_proxy_handler (jerry_value_t proxy_value);
+```
+
+- `proxy_value` - Proxy object value
+- return value
+  - type error - if proxy_value is not a Proxy object
+  - handler object - otherwise
+
+*New in version 2.4*.
```

Comment:
Is this really 2.4 version? AFAIK this is NEXT_VERSION (or the one we use for new unreleased versions)